tests/test_images_embed.py

```diff
@@ -88,37 +89,37 @@ def test_load_embed_data(images_data):
 
 def test_load_vector(svg_image):
     """Test correct loading of SVG images."""
-    embedder = embed._EmbedImages()
+    embedder = embed._EmbedImages(os.path.join(config.DIR_TEMP, 'images'))
     node = embedder.load_vector(svg_image)
     assert getattr(node, 'name', None) == 'svg'
 
 
 def test_embed_vector(svg_image):
     """Test embedding of SVG image."""
-    embedder = embed._EmbedImages()
+    embedder = embed._EmbedImages(os.path.join(config.DIR_TEMP, 'images'))
     soup = bs4.BeautifulSoup('<img src="foo.svg">', features='lxml')
     assert soup.svg is None
     embedder.embed_vector(soup.img, svg_image)
     assert soup.svg is not None
 
 
-def test_embed_images(html_file, mocker):
+def test_embed_images(html_file, mocker, tmp_path):
     """Test general method for embedding images."""
     mocker.patch('src.images.embed._EmbedImages.embed_vector', mocker.Mock())
-    embedder = embed._EmbedImages()
+    embedder = embed._EmbedImages(os.path.join(config.DIR_TEMP, 'images'))
     image = '/images/foo.svg'
     embedder.embed_images(html_file, {image})
     args = embedder.embed_vector.call_args[0]
     assert len(args) == 2
     assert args[1] == config.DIR_TEMP + image
 
 
-def test_run(svg_image, html_file, images_data):
+def test_run(svg_image, html_file, images_data, tmp_path):
```

Comment:
Falta el docstring
podría ser
"""Test API's method to embed the image in the file."""

Reply:
Agregado, gracias.